source/Memcached-Tests/MDProtocolTest.class.st

```diff
@@ -2,35 +2,40 @@
 Unit tests for MDProtocol
 "
 Class {
-	#name : #MDProtocolTest,
-	#superclass : #TestCase,
+	#name : 'MDProtocolTest',
+	#superclass : 'MDTest',
 	#instVars : [
 		'protocol',
 		'data',
 		'key'
 	],
-	#category : #'Memcached-Tests'
+	#category : 'Memcached-Tests',
+	#package : 'Memcached-Tests'
 }
 
-{ #category : #running }
+{ #category : 'running' }
 MDProtocolTest >> setUp [
 
-	protocol := MDProtocolBuilder buildDefault.
-	key := ByteArray with: 107 with: 101 with: 121.
-	data := ByteArray
-		with: 49
-		with: 51
-		with: 51
-		with: 55
+	super setUp.
+	protocol := self
+		            withCIHostNameDo: [ :hostname |
+			            MDProtocolBuilder new
+				            host: hostname;
+				            port: MDProtocolBuilder defaultPort;
+				            build
+			            ]
+		            else: [ MDProtocolBuilder buildDefault ].
+	key := #[ 107 101 121 ].
```

Comment:
I remember changing the code generator to avoid these literals for making it more portable. For instante to VA Smalltalk